### PR TITLE
Competition revamped ui

### PR DESCRIPTION
**Pull request by dcorisis**
added competition features and top three buildings

### CODE REVIEW

**Comment by tjdfalcon** [comment]
Can the log-in stuff be removed from this pull request?  I believe @dkaai was implementing some stuff that will need to be reconciled.

**Comment by dkaai** [comment]
Looking at the files changed, everything should be good.  The only potential problem is in loginDisplay.html.  I'm using the same file, but have added a few things to it, but it should be okay.

**Comment by dkaai** [comment]
When looking at the top buildings tab and switching to full view, the graph formatting is wrong for the first competition you look at.  If there is no data for a competition yet, the top buildings tab displays the top buildings from the most recently selected competition that has data.  On my computer, the formatting of some the building names in the panels for the top buildings tab are off.  For example, in The Longest Names competition, the second place name (Theatre/Dance Rehearsal) doesn't really look right.

**Comment by tjdfalcon** [comment]
I would double-check that some changes are not getting lost, particularly in the html and css.

File: client/app/scripts/controllers/competitionGraph.js [hunk 1]
```diff
@@ -40,15 +51,14 @@ angular.module('clientApp')
             calcAllChanges();
             sortChanges();
             createData();
+            compEditSvc.setTopThree([$scope.data[0].key, $scope.data[1].key, $scope.data[2].key]);
```
Comment (tjdfalcon) [inline client/app/scripts/controllers/competitionGraph.js]: This line is throwing errors sometimes.

**Comment by tjdfalcon** [comment]
Also, are we setting backgrounds based on the view or based on the time or both?

**Comment by tjdfalcon** [comment]
:100:

**Pull request merged**